clients/src/test/java/org/apache/kafka/common/config/AbstractConfigTest.java

```diff
@@ -599,6 +624,31 @@ public TestConfig(Map<?, ?> props) {
         }
     }
 
+    private static class InterceptorTestConfig extends AbstractConfig {
+        private final int targetInterceptor = 3;
+        private static final ConfigDef CONFIG;
+        private static final String INTERCEPTOR_CLASSES_CONFIG_DOC = "A list of classes to use as interceptors.";
+
+        public static final String INTERCEPTOR_CLASSES_CONFIG = "interceptor.classes";
+        public static final String CLIENT_ID_CONFIG = "client.id";
+        public static final String BOOTSTRAP_SERVERS_CONFIG = "bootstrap.servers";
+
+        static {
+            CONFIG = new ConfigDef().define(INTERCEPTOR_CLASSES_CONFIG,
+                    Type.LIST,
+                    Collections.emptyList(),
+                    Importance.LOW,
+                    INTERCEPTOR_CLASSES_CONFIG_DOC);
+        }
+        public InterceptorTestConfig(Map<?, ?> props) {
+            super(CONFIG, props);
+        }
+
+        public int getTargetInterceptor() {
+            return targetInterceptor;
+        }
+    }
```

Comment:
Sorry, I'm still not in favor of this additional class. The name of the config property isn't super critical and it's not necessary to try to instantiate metrics reporters when using the `metric.reporters` property in the test config; this is only constructed for testing, after all.

Please remove this class and simplify the testing logic by using the existing `TestConfig` class.

Reply:
@C0urante No apologies necessary.  I've reverted the code back to your preference.